procedures/release-duty/index.rst

```diff
@@ -63,24 +65,23 @@ Email
 
 As ReleaseDuty you need to *subscribe* to certain mailing lists:
 
-*  All types of sign-offs and approvals should go to `release signoff mailing list <https://mail.mozilla.org/listinfo/release-signoff>`__
-*  All releng automation emails should go to `release-automation-notifications <https://groups.google.com/a/mozilla.com/forum/?hl=en#!forum/release-automation-notifications>`__
-*  All discussion topics should go to `release drivers mailing list <https://mail.mozilla.org/listinfo/release-drivers>`__
+ * All types of sign-offs and approvals should go to `release signoff mailing list <https://mail.mozilla.org/listinfo/release-signoff>`__
+ * All discussion topics should go to `release drivers mailing list <https://mail.mozilla.org/listinfo/release-drivers>`__
 
 These other mailing lists will likely have useful discussions and information:
-* `RelEng internal mailing list <release@mozilla.com>`__
-* `Thunderbird release drivers mailing list <https://mail.mozilla.org/listinfo/thunderbird-drivers>`__
-* `release-automation-notifications-thunderbird mailing list <https://mail.mozilla.org/listinfo/release-automation-notifications-thunderbird>`__
-* (optional) `release-automation-notifications-dev mailing list <https://groups.google.com/a/mozilla.com/forum/#!forum/release-automation-notifications-dev>`__
+ * `RelEng internal mailing list <release@mozilla.com>`__
```

Comment:
This reminds me we should add you all on this ML, I don't think you're currently there.